packages/react/src/QuestionnaireForm/QuestionnaireFormComponents/QuestionnaireFormGroup.tsx

```diff
@@ -14,6 +14,10 @@ interface QuestionnaireRepeatableGroupProps {
 export function QuestionnaireRepeatedGroup(props: QuestionnaireRepeatableGroupProps): JSX.Element | null {
   const [responses, setResponses] = useState(props.response);
 
+  useEffect(() => {
+    setResponses(props.response);
```

Comment:
We should check with @ThatOneBro  on this - It might be fine, but I'm not sure what our philosophy is here in this component, whether `responses` is intended to be a "controlled" component or not.

Reply:
I had an issue with nested items. 
When questions belonged to the same group/ .item. It would work. 
But I nested several questions in different groups, the calculated values would not refresh in the UI.
useEffect would solve the issue in the QuestionnaireRepeatedGroup would solve the issue.

Reply:
I do remember when this was originally being worked on that the intent was to propagate all changes to the root via `props.onChange` and essentially enforce the hierarchy all being controlled components... whether that is the case today or not, I'm not sure. Should probably be something we review in the near future.

This change seems sane to me at first glance though, seems like it might have been a bug that has been there